dev/fattest.simplicity/src/componenttest/topology/database/container/DatabaseContainerUtil.java

```diff
@@ -73,7 +77,23 @@ public static void setupDataSourcePropertiesForCheckpoint(LibertyServer serv, Jd
         //Get libraries to be changed
         Map<String, Fileset> libraries = getLibraries(cloneConfig);
         //Modify those datasources
-        modifyDataSourcePropsForDatabase(datasources, authDatas, libraries, cloneConfig, serv, cont);
+        modifyDataSourcePropsForDatabase(datasources, authDatas, libraries, cloneConfig, serv, cont, true);
+    }
+
+    public static void setupDataSourceEnvForCheckpoint(LibertyServer serv, JdbcDatabaseContainer<?> cont) throws Exception {
+        java.util.Properties envProps = serv.getServerEnv();
+        envProps.put(DB_DRIVER, DatabaseContainerType.valueOf(cont).getDriverName());
+        boolean isDerby = DatabaseContainerType.valueOf(cont) == DatabaseContainerType.Derby || DatabaseContainerType.valueOf(cont) == DatabaseContainerType.DerbyClient;
+        if (!isDerby) {
+            envProps.put(DB_HOST, cont.getHost());
+            envProps.put(DB_PORT, Integer.toString(cont.getFirstMappedPort()));
+            envProps.put(DB_USER, cont.getUsername());
+            envProps.put(DB_PASSWORD, cont.getPassword());
+        }
```

Comment:
```suggestion

```

We don't need to worry about the connection properties.  That is already taken care of when we `modifyDataSourcePropsForDatabase` is called.